sklearn/feature_extraction/dict_vectorizer.py

```diff
@@ -312,7 +312,9 @@ def get_feature_names(self):
         return self.feature_names_
 
     def restrict(self, support, indices=False):
-        """Restrict the features to those in support.
+        """Gives support for feature selection by restricting the features to those in support.
```

Comment:
Everything should be less than 80 characters according to pep8. The first line of the docstring should be a single line, so please rather reformulate.